test/server/cardImplementations/01_SOR/ViperProbeDroid.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Viper Probe Droid', function() {
+    integration(function() {
+        describe('Viper Probe Droid\'s Ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['viper-probe-droid']
+                    },
+                    player2: {
+                        hand: ['wampa', 'battlefield-marine', 'pyke-sentinel']
```

Comment:
Would be good to add some extra units in the opponent's arena(s) so we have some negative results to be sure the ability is restricted to cards from hand